test/io/configs/expected/no-defaults.config

```diff
@@ -35,6 +35,7 @@ server-trace-header = "X-Request-Id"
 server-timing-enabled = true
 server-unix-socket = "/tmp/pgrst_io_test.sock"
 server-unix-socket-mode = "777"
+admin-server-host = "!4"
```

Comment:
In this test, the values should be different than the default (different from `"!4"`).

Reply:
Thanks, done